llvm/lib/Target/SPIRV/SPIRVGlobalRegistry.cpp

```diff
@@ -390,6 +390,26 @@ SPIRVGlobalRegistry::getOrCreateConsIntArray(uint64_t Val,
                                        LLVMArrTy->getNumElements());
 }
 
+Register
+SPIRVGlobalRegistry::getOrCreateConsNullPtr(MachineIRBuilder &MIRBuilder,
```

Comment:
s/Cons/Const

Reply:
Ok. BTW we already have several `getOrCreateConsInt*` names in SPIRVGlobalRegistry...